event/parsing/totalBootTimeParser.go

```diff
@@ -0,0 +1,170 @@
+/**
+ *  Copyright (c) 2020  Comcast Cable Communications Management, LLC
+ */
+
```

Comment:
This looks like the wrong copyright notice.  I'm not sure if this is everywhere in the repo; if it's everywhere, this can be fixed in a separate pr.

Reply:
Made an issue for it